src/Infrastructure/ErrorLoggingFileFetcher.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Infrastructure;
+
+use FileFetcher\FileFetcher;
+use FileFetcher\FileFetchingException;
+use Psr\Log\LoggerAwareInterface;
+use Psr\Log\LoggerInterface;
+
+class ErrorLoggingFileFetcher implements FileFetcher, LoggerAwareInterface {
+
+	private $wrappedFileFetcher;
+
+	private $logger;
+
+	public function __construct( FileFetcher $fileFetcher, LoggerInterface $logger ) {
+		$this->wrappedFileFetcher = $fileFetcher;
+		$this->logger = $logger;
+	}
+
+	public function fetchFile( $fileUrl ) {
+		try {
+			return $this->wrappedFileFetcher->fetchFile( $fileUrl );
+		} catch ( FileFetchingException $e ) {
+			$this->logger->error( $e->getMessage(), [
+				'code' => $e->getCode(),
+				'trace' => $e->getTraceAsString()
+			] );
+			return '';
```

Comment:
The reasoning behind this is as follows: We want to have some kind of notification when the files are missing, but this error condition is not not serious enough to bring down the whole application with an exception. And it's not an error we can test against because it's a server configuration thing. 

I could do a try...catch in the WordListFileReader instead, but then I'd have to pass it a logger, which in my view is not a proper separation of concerns.

The previous code did exactly the same, only it did log a notice, which was dropped, so were never aware that our files were missing.